blocks/blocks.go

```diff
@@ -20,47 +20,47 @@ type Block interface {
 }
 
 // Block is a singular block of data in ipfs
-type RawBlock struct {
+type BasicBlock struct {
```

Comment:
Note, that I originally named this RawBlock in commit b84cbec2.  Both commits where factored out of #2634.  The name `RawBlock` was only a temporary name, I meant to squash this in b84cbec2 but messed up the rebase and didn't notice until after it was merged.